### DISCUSSION TITLE
历史版本镜像如何下载

### DISCUSSION THREAD

**Discussion by Liuzq666** [secretflow/secretflow#1361] [General]
怎么找到历史版本的各个安装包啊，切换文档版本下载链接好像还是最新版本的链接

**Comment by zimu-yuxi** [comment]
1.allinone包目前已经支持文档切换版本，下载历史版本安装包
2.隐语相关镜像可以指定历史版本pull

**Reply by john8628** [reply]
allinone切换到1.7.0下载的还是最新的版本；

**Reply by zimu-yuxi** [reply]
复制1.8.0的下载地址，修改版本号
